premailer/premailer.py

```diff
@@ -140,7 +140,7 @@ def __init__(
         disable_link_rewrites=False,
         preserve_internal_links=False,
         preserve_inline_attachments=True,
-        exclude_pseudoclasses=True,
+        exclude_pseudoclasses=False,
```

Comment:
This makes me nervous. Most people use premailer programmatically rather that the `__main__.py` endpoint. A change here is expected to break the tests. (At the time of writing I'm waiting for Travis to finish) And potentially it might change peoples implementations when they upgrade to the next version. 
Which is right I'm not sure about but I'm sure we should have the same defaults in `__main__`, this file and in the README.